modules/nf-scil/preproc/n4/meta.yml

```diff
@@ -0,0 +1,58 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "preproc_n4"
+description: Bias field correction using N4
+keywords:
+  - "ANTs"
+    description: ANTs computes high-dimensional mappings to capture the statistics of brain structure and function.
+
+  - "scilpy"
+    description: Scilpy is the main library supporting research and development at the Sherbrooke Connectivity Imaging Lab
+tools:
+  - "preproc":
```

Comment:
Remove "preproc". It is autogenerated based on the category we specify, but we don't want it